src/tiled/layerdock.h

```diff
@@ -77,6 +80,22 @@ private slots:
 };
 
 /**
+ * Delegate for drawing an eye icon in LayerView when the layer is visible.
+ */
+class LayerVisibilityDelegate: public QItemDelegate
+{
+public:
+    explicit LayerVisibilityDelegate(QObject *parent = 0);
```

Comment:
I would prefer if you typed `nullptr` instead of `0` here. Tiled already depends on C++11.